### DISCUSSION TITLE
Feature Request: Open External Links in New Tab

### DISCUSSION THREAD

**Discussion by ghost** [thegeeklab/hugo-geekdoc#691] [Ideas]
Thanks so much for making this theme !

**Proposal:** Add a configuration flag in `hugo.toml` for opening external links in a new tab.

**Comment by xoxys** [comment]
Thanks for your request. Personally, I strongly believe that this should be controlled by the user, not the website. Anyway, if more users would like to have this feature, we can take a closer look.
